### DISCUSSION TITLE
zsh: command not found: brew

### DISCUSSION THREAD

**Discussion by whoisgrayson** [Homebrew/discussions#5029] [Everyday usage]
I have installed Homebrew and am now trying to install Folderify by using this:

brew install folderify

but it keeps saying

zsh: command not found: brew

Anyone know what I am doing wrong? Thanks!

**Comment by DanielKehoe** [comment]
This is asked often so I wrote an article [Zsh: command not found: brew](https://mac.install.guide/homebrew/zsh-command-not-found-brew.html). I hope it's useful. Comment if you think the article can be improved. Basically, here are possible reasons:
- You need to restart your Terminal.
- Homebrew might not be not installed.
- Your $PATH setting is missing or incorrect.
The article gives instructions.

**Comment by SMillerDev** [comment]
Did you follow the instructions about `brew shellenv` at the end of the install?
